contracts/call-number/Cargo.toml

```diff
@@ -35,4 +35,5 @@ wasmer-types = { version = "1.0.2", features = ["enable-serde"] }
 [dev-dependencies]
 cosmwasm-schema = { path = "../../packages/schema" }
 cosmwasm-vm = { path = "../../packages/vm", default-features = false, features = ["iterator"] }
-wasmer-types = { version = "1.0.2"}
+wasmer = { version = "=2.2.1", default-features = false, features = ["cranelift", "universal", "singlepass"] }
+wasmer-types = "=2.2.1"
```

Comment:
nit: No Newline at End of File

Reply:
Fixed, please could you check.
If ok, please resolved.